hotfixes-rhel-coreos-9.yaml

```diff
@@ -0,0 +1,5 @@
+hotfixes:
+  - osmajor: "8"
+    repo: rhel-8.6-appstream
```

Comment:
8.6 here but you mention 8.8? Isn't this RPM more likely to be a custom rhaos build instead?

Reply:
Yeah, this is just a demonstration change.  That said, I do think it'd likely make sense to get the change directly into 8.6.z too.